CMakeLists.txt

```diff
@@ -24,6 +24,13 @@ set(CMAKE_CXX_STANDARD 17)
 
 project(rpp CXX)
 
+project(rpp VERSION 1.4.0)
+
+if(${ROCM_PATCH_VERSION})
+   set(PROJECT_VERSION_PATCH ${ROCM_PATCH_VERSION})
+   set(PROJECT_VERSION "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}")
+endif()
+
 # RPP Version
 set(VERSION "1.4.0")
```

Comment:
@arvindcheru can we use the version info from one place?

Reply:
Updated in latest version